app/src/main/java/com/andela/omokolataiwo/levelup/view/MainActivity.java

```diff
@@ -32,20 +35,40 @@
      */
     private List<GithubProfile> githubProfilesParcel;
 
-    /**
-     * github profile presenter.
-     */
-    private final GithubProfilePresenter githubProfilePresenter =
-            new GithubProfilePresenter(this);
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
         recyclerView = findViewById(R.id.recycler_view);
+        final SwipeRefreshLayout swipeRefreshLayout = findViewById(
+                R.id.main_activity_swipe_refresh);
+
+        final GithubProfilePresenter githubProfilePresenter = new GithubProfilePresenter(this,
+                (CoordinatorLayout) findViewById(R.id.coordinatorLayout));
+
+        swipeRefreshLayout.setColorSchemeResources(
```

Comment:
@omokolataiwo 
It is advisable to reduce the number of lines of code presenter in the `onCreate` method.
So...instead, you can implement those lines of code in a separate method, then reference(call) the method in the `onCreate` method
Try extracting the `swipe to refresh codes` to a separate method, then call it in the `onCreate` method.